src/WrappedMToken.sol

```diff
@@ -443,9 +463,26 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
         emit Claimed(account_, claimRecipient_, yield_);
         emit Transfer(address(0), account_, yield_);
 
-        if (claimRecipient_ != account_) {
+        uint240 yieldNetOfFees_ = yield_;
+
+        if (accountInfo_.hasEarnerDetails) {
```

Comment:
consider (related to previous comment)
```suggestion
        if (!isTTGEarner && accountInfo_.hasEarnerDetails) {
```

Reply:
Aside from the unnecessary "TTG", if we were to check if an account is a registrar earner first, then it would result in two queries to the registrar since getting the details from the earner manager also queries a registrar for this information.

I think a better solution would be to allow start earning for to be called again for an account in order to set the has earner details property to false. 

Or perhaps have this internal claim function set the property to false once it sees there is no longer an admin when querying the earner details.

Reply:
```I think a better solution would be to allow start earning for to be called again for an account in order to set the has earner details property to false.``` good choice